packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -496,9 +496,18 @@ The following sections outline the key event types that are captured by the syst
 
 | Event                                | Description                                                                                                                  |
 | :-------------------                 | :----------------------------------                                                                                          |
-| system.org.rate_limit.warning        | An endpoint is near its [rate limit](/docs/reference/rate-limits/).                                                 |
-| system.org.rate_limit.violation      | An endpoint exceeds its [rate limit](/docs/reference/rate-limits/).                                            |
-| core.concurrency.org.limit.violation | A request exceeds the org's allotted [concurrency limit](/docs/reference/rate-limits/#concurrent-rate-limits). |
+| `system.org.rate_limit.warning`        | An endpoint is near its [rate limit](/docs/reference/rate-limits/).                                                 |
+| `system.org.rate_limit.violation`      | An endpoint exceeds its [rate limit](/docs/reference/rate-limits/).                                            |
+| `core.concurrency.org.limit.violation` | A request exceeds the org's allotted [concurrency limit](/docs/reference/rate-limits/#concurrent-rate-limits). |
+| `system.operation.rate_limit.violation`        | A request exceeds a [rate limit](/docs/reference/rate-limits/).                                                 |
+| `system.operation.rate_limit.warning`        | An endpoint has reached a significant portion of its [rate limit](/docs/reference/rate-limits/) within a period.                                                 |
+| `system.operation.rate_limit.notification`        | Additional information is provided about the rate limiting decisions when an endpoint has exceeded its [rate limit](/docs/reference/rate-limits/).                                                 |
```

Comment:
if we decide against @peteryaworsky-okta 's suggestion above, these hyperlinks should point to a page about the Operation Rate Limits specifically

Reply:
Went with @peteryaworsky-okta 's suggestion, therefore no further hyperlinks needed.